### PR TITLE
feat: add a --no-color option

### LINKED ISSUE #631: Add a --no-color option
Most CLIs offer a --no-color option to avoid having messages containing color codes. 

Here's an example with a simple Rust system call:
```Rust
stderr: "\u{1b}[1m\u{1b}[31m[ERROR] \u{1b}[39m\u{1b}[22mYou do not belong to this organisation so you cannot access it\n"
```

NOTE: As a temporary fix, you can export FORCE_COLOR=0 to disable the color.

### PR DESCRIPTION
**Pull request by davlgd**
Fix #631

### CODE REVIEW

File: bin/clever.js [hunk 1]
```diff
@@ -12,6 +12,13 @@ if (process.argv.includes('--autocomplete-index')) {
   process.env.CLEVER_QUIET = '1';
 }
 
+// These need to be set before other stuff
+// Disable color globally if the user asked to
+if (process.argv.includes('--nocolor')) {
```
Comment (aurrelhebert) [inline bin/clever.js:17]: I think that `no-color` will be better suited. Most of other available option uses a `-` to split words, as for example `--no-update-notifier`.

**Review by aurrelhebert — COMMENTED** [review]
LGTM, it's working fine for me

File: bin/clever.js (hunk 1 above)
Comment (davlgd) [reply bin/clever.js:17]: It wasn't working as the parser was seeing 2 different options (if I remember weel). It's why I removed the middle '-'

Comment (davlgd) [reply bin/clever.js]: I looked at why I got this behavior, as sometimes the `-` was OK, but not with this option. After several attempts and further analysis, I found that the issue occurs only if an option starts with `--no-`.

For example:

```
$ clever --no-color
Unknown option: color
```

```
$ clever --color-no
Unknown option: color-no
```

For the same reason, the `--no-update-notifier` option can't be used:

```
$ clever logs --no-update-notifier
Unknown option: update-notifier
```

I thought it was a bug/feature of cliparse, but after more deep dive I discovered it's a feature of the `minimist` package used by cliparse. If an argument starts with `--no-` this part is not included in its name and its value is considered as false. You can check that in `minimist` [parsing tests](https://github.com/minimistjs/minimist/blob/main/test/parse.js) and [code](https://github.com/minimistjs/minimist/blob/fa422e82ce6d06278a515343670ebdc161417b90/index.js#L169).

So I pushed an update to deal with this. Now, there is a `--color` option, `true` by default. The user can also use `--no-color` as a shorter form.

**Comment by davlgd** [comment]
ops team asked to implement a quite generic approach for this kind of option: 
- if the app is launched in a non-tty environment: no-color by default
- if the app is launched in a tty environment: color by default
- if the user asks a particular mode, it is given priority

**Review by hsablonniere — APPROVED** [review]
LGTM :+1:

**Pull request merged**